tilecloud_chain/server.py

```diff
@@ -394,6 +394,7 @@ def serve(self, path, params, **kwargs):
                 'Cache-Control': "max-age={}".format((3600 * self.expires_hours)),
                 'Access-Control-Allow-Origin': '*',
                 'Access-Control-Allow-Methods': 'GET',
+                'TileBackend': 'Cache',
```

Comment:
It would be more "standard" to name it `Tile-Backend`.

Reply:
+1 :-)